docs/module.md

```diff
@@ -210,15 +210,15 @@ Pipeline B
 └── main.nf
 ```
 
-With the possibility to keep the template files inside the project L, A and B can use the modules defined in L without any changes. A future project C would do the same, just cloning L (if not available on the system) and including its module script.
+With the possibility to keep the template files inside the project L, A and B can use the modules defined in L without any changes. A future project C would do the same, just cloning L (if not available on the system) and including its module.
```

Comment:
This is hard to follow. The relationship between Project L, and script A and B isn't clear.

Reply:
We can address this in a separate PR if you'd like. I mainly wanted to clean up some vocabulary here, don't really want to re-work this example in this PR